src/constants/api.jsx

```diff
@@ -1,4 +1,4 @@
-const API_LINK: string = "https://api.fda.gov"
+const API_LINK: string = "https://openfda-api.preprod.fda.gov"
```

Comment:
@beardedfinch

Reply:
I have those this change and the same one for the explorers prestaged, just waiting to push in case they want more changes to the demo site.